virtcontainers/pkg/oci/utils.go

```diff
@@ -509,6 +509,13 @@ func addHypervisorConfigOverrides(ocispec specs.Spec, config *vc.SandboxConfig,
 		}
 	}
 
+	if value, ok := ocispec.Annotations[vcAnnotations.PFlash]; ok {
+		if value != "" {
+			flash := strings.Split(value, ",")
```

Comment:
yeah, it will lead to bug, but I have no idea to avoid that. do you have any idea?

Reply:
I'd use a space as separator and mention in the annotations document that the pflash file path **must not** contains any space

Reply:
thanks @devimc -, that's a good idea.
@jodh-intel -, WDYT ?

Reply:
Well, files can contain spaces too ;) Ideally, this would be possible:

```json
"annotations": {
    "pflashes" : ["/foo/bar", "/baz/wibble", "/a/b/c", "/the/end"]
}
```

... but that is illegal as the OCI spec states that annotation [values MUST be strings](https://github.com/opencontainers/runtime-spec/blob/master/config.md#annotations).

You *could* encode the array of paths as follows but that's starting to get a bit baroque imho:

```json
"annotations": {
    "pflashes" : "[\"/foo/bar\", \"/baz/wibble\", \"/a/b/c\", \"/the/end\"]"
}
```

I think @devimc is right - the best approach is probably to choose a delimiter, document it clearly, and then be careful to reject any pflash file paths that don't exist.

Reply:
thanks @jodh-intel -, so what about choosing "," as the delimiter (as it is clearer than space) and document for it . Also check the pathname after split.

Reply:
We're back to the start :smile:

I'm not 100% happy with the approach, but I think that due to the OCI restrictions that is the best we can do. So yes, let's do that :smile:

Reply:
thanks @jodh-intel

Reply:
hi @jodh-intel , I clean these codes, as there maybe no need of those.